admin_guide/manage_users.adoc

```diff
@@ -9,127 +9,103 @@
-    "_UID": "0",
-}
-----
-====
+[[managing-users-viewing-user-and-identity-lists]]
+== Viewing User and Identity Lists
 
-This error is generated because OpenShift sees the user data in the internal
-database that does not correspond to the external identity.
+{product-title} user configuration is stored in several locations within
+{product-title}. Regardless of the identity provider, {product-title} internally
+stores details like role-based access control (RBAC) information and group
+membership. To completely remove user information, this data must be removed in
+addition to the user account.
 
-In OpenShift, two object types contain user data outside the identification
-provider: type user and type identity.
+In {product-title}, two object types contain user data outside the
+identification provider: `*user*` and `*identity*`.
 
-Type user:
+To get the current list of users:
 
 ====
 ----
-$ oc get user ashley
+$ oc get user
 NAME      UID                                    FULL NAME   IDENTITIES
-peter     75e4b80c-dbf1-11e5-8dc6-0e81e52cc949               htpasswd_auth:ashley
+demo     75e4b80c-dbf1-11e5-8dc6-0e81e52cc949               htpasswd_auth:demo
 ----
 ====
 
-Type identity (from the user record):
+To get the current list of identities:
 
 ====
 ----
-oc get identity htpasswd_auth:ashley
+oc get identity
 NAME                  IDP NAME        IDP USER NAME   USER NAME   USER UID
-htpasswd_auth:ashley  htpasswd_auth   ashley          ashley      75e4b80c-dbf1-11e5-8dc6-0e81e52cc949
+htpasswd_auth:demo    htpasswd_auth   demo            demo        75e4b80c-dbf1-11e5-8dc6-0e81e52cc949
 ----
 ====
 
 Note the matching UID between the two object types. If you attempt to change the
-authentication provider after starting to use OpenShift, the user names that
-overlap will not work because of the entries in the identity list, which will
-still point to the old authentication method.
-
-=== Properly Deleting a User
-
-To delete a user, all three sources of the user must be removed:
-
-. Identification manager
-. User record
-. Identity record
-
-Once you complete these steps, a new account will be created in OpenShift
-when the user logs in again.
-
-==== Deleting the Identification Manager Entry
-
-If you are using `htpasswd`, delete the entry in the *_htpasswd_* file that is
-configured for OpenShift with the username and password.
+authentication provider after starting to use {product-title}, the user names
+that overlap will not work because of the entries in the identity list, which
+will still point to the old authentication method.
 
-For external identification management like Lightweight Directory Access
-Protocol (LDAP) or Internet Download Manager (IDM), use the user management
-tools to remove the user entry.
-
-==== Deleting the User Record
+[[managing-users-deleting-a-user]]
+== Deleting a User
 
-Delete the user record with `oc delete user`:
+To delete a user:
 
+. Delete the user record with `oc delete user`:
```

Comment:
I could see dropping "with `oc delete user`", since we show the command immediately following. Either way.